### ISSUE TITLE
NameError: name 'N' is not defined when setting thresholds for all nodes

### ISSUE THREAD

**Issue by chengjun** [benmaier/thresholdmodel#1]
```Python
from thresholdmodel import ThreshModel

N = 1000
k = 10
initially_activated = np.arange(100)
G = nx.fast_gnp_random_graph(N, k/(N-1.0))

thresholds = np.random.rand(G.number_of_nodes()) 


Thresh = ThreshModel(G, initially_activated, thresholds)
t, cascade_size = Thresh.simulate()

```

```python
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-43-929e4fe7e852> in <module>
      9 
     10 
---> 11 Thresh = ThreshModel(G, initially_activated, thresholds)
     12 t, cascade_size = Thresh.simulate()

/opt/anaconda3/lib/python3.7/site-packages/thresholdmodel/model.py in __init__(self, G, initially_activated, thresholds, weight)
    127 
    128         self.set_initially_activated(initially_activated)
--> 129         self.set_thresholds(thresholds)
    130 
    131     def set_thresholds(self,thresholds):

/opt/anaconda3/lib/python3.7/site-packages/thresholdmodel/model.py in set_thresholds(self, thresholds)
    154 
    155         else:
--> 156             assert(len(thresholds) == N)
    157             assert(np.all(thresholds <= 1) and np.all(thresholds>=0))
    158 

NameError: name 'N' is not defined
```

**Comment by benmaier** [comment]
should be fixed in [v0.0.3](https://github.com/benmaier/thresholdmodel/releases/tag/v0.0.3)

**Comment by chengjun** [comment]
Thanks. The problem has been fixed in v0.0.3.
